src/panels/KaitoModelsPanel.ts

```diff
@@ -96,21 +100,102 @@ export class KaitoModelsPanelDataProvider implements PanelDataProvider<"kaitoMod
     nullIsFalse(value: boolean | null): boolean {
         return value ?? false;
     }
+    parseGPU(gpuRequirement: string): [string, number] {
+        const regex = /^Standard_NC(\d+)(ads_A100_v4|s_v3)$/;
+        const match = gpuRequirement.match(regex);
+        if (match) {
+            const cpus = parseInt(match[1], 10);
+            let family: string;
+            switch (match[2]) {
+                case "ads_A100_v4":
+                    family = "StandardNCADSA100v4Family";
```

Comment:
💡 if this is too many times use, we can `enum` `case` types like `ads_A100_v4` et. al. for better handling.